packages/bot-web-ui/src/components/dashboard/bot-builder/toolbar/__tests__/workspace-group.spec.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { mockStore, StoreProvider } from '@deriv/stores';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { render, screen } from '@testing-library/react';
+import { mock_ws } from 'Utils/mock';
+import { DBotStoreProvider, mockDBotStore } from 'Stores/useDBotStore';
+import WorkspaceGroup from '../workspace-group';
+import userEvent from '@testing-library/user-event';
+
+jest.mock('@deriv/bot-skeleton/src/scratch/blockly', () => jest.fn());
+jest.mock('@deriv/bot-skeleton/src/scratch/dbot', () => ({
+    saveRecentWorkspace: jest.fn(),
+    unHighlightAllBlocks: jest.fn(),
+}));
+
+describe('WorkspaceGroup', () => {
+    let wrapper: ({ children }: { children: JSX.Element }) => JSX.Element;
+    const mock_props = {
+        has_redo_stack: true,
+        has_undo_stack: true,
+        onResetClick: jest.fn(),
+        onSortClick: jest.fn(),
+        onUndoClick: jest.fn(),
+        onZoomInOutClick: jest.fn(),
+        toggleLoadModal: jest.fn(),
+        toggleSaveModal: jest.fn(),
+    };
+
+    beforeAll(() => {
+        const mock_store = mockStore({});
+        const mock_DBot_store = mockDBotStore(mock_store, mock_ws);
+
+        wrapper = ({ children }: { children: JSX.Element }) => (
+            <StoreProvider store={mock_store}>
+                <DBotStoreProvider ws={mock_ws} mock={mock_DBot_store}>
+                    {children}
+                </DBotStoreProvider>
+            </StoreProvider>
+        );
+    });
+
+    it('should render WorkspaceGroup in desktop', async () => {
```

Comment:
@vinu-deriv i think we dont need async here same for all the below ones 🙏

Reply:
will remove